Esri/arcgis-runtime-samples-ios
# [Fix] ODR tags and redundant shapefile removal

## Description

This PR fixes the issue brought by #1134 

- It used Aurora, CO GeoPackage and LA Trails Geodatabase, but didn't include them in the dependency plist entry
- It removed the sample `Feature layer (shapefile)` that uses `Public_Art.shp`, but didn't remove the resource files which are now unused.

Currently `Display feature layers` sample crashes in the App Store version (but not the local build) because both resources are not downloaded as ODR when the sample loads.

## Linked Issue(s)

- `common-samples/issues/3425`
- `common-samples/issues/3407`

## How To Test

After this PR is merged and subsequently uploaded to App Store Connect for release or TestFlight, we shall verify that `Display feature layers` is working as expected.
